compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1227,7 +1227,18 @@ static LogicalResult setDefaultGenericOpRootConfig(
                                  parallelTileSizes, reductionTileSizes);
 
   TileSizesListType tileSizes;
-  tileSizes.push_back(flowTileSizes);
+  SmallVector<int64_t> updatedFlowTileSizes;
```

Comment:
Please add doc about why we are doing this

Reply:
Done